### PR TITLE
Configure circleci config

### PR DESCRIPTION
**Pull request by QuentinBisson**
Towards https://github.com/giantswarm/giantswarm/issues/23238

Splitting of https://github.com/giantswarm/heartbeatctl/pull/3 part 1

### CODE REVIEW

File: tools.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build tools
```
Comment (TheoBrigitte) [inline tools.go:1]: What is this line for ?

**Review by TheoBrigitte — APPROVED** [review]
Looks good, thanks for splitting this.

File: tools.go (hunk 1 above)
Comment (QuentinBisson) [reply tools.go:1]: This lines tells go that the imports in this file are not to be added to the go modules as they are build dependencies.
This file had // +build tools earlier so go generate can work but circleci requires //go:build tools

Comment (TheoBrigitte) [reply tools.go:1]: I think this might be because we might have different go version here and in CI

**Review by TheoBrigitte — APPROVED** [review]

**Pull request merged**